.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,117 @@
+# This is a reusable workflow for running ShellCheck,
+# a linter for shell scripts (https://shellcheck.net).
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+on: 
+  workflow_call:
+    inputs:
+
+      exclude-patterns:
+        type: string
+        required: false
+        default: ""
+        description: |
+          File paths matching these patterns will be skipped when running ShellCheck.
+          Space-separated. Patterns containing spaces are not supported.
+          Wildcard asterisks (*) are supported.
+          Note that all paths are prefixed with './', so in order to exclude a path,
```

Comment:
This is a little confusing, do you mean that they automatically get prefixed and you don't have to do anything or that all paths must start with either `./` or `*` ?

Reply:
The latter. You're right, it's confusing, I tried again. If you still find it confusing then I can definitely make a followup PR.